shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -714,6 +714,9 @@ - (void)goToApplicationLifecycle:(nonnull NSString*)state {
       return flutter::PointerData::Change::kUp;
     case UITouchPhaseCancelled:
       return flutter::PointerData::Change::kCancel;
+    default:
+      // TODO(53695): Handle the `UITouchPhaseRegion`... enum values.
```

Comment:
Can we print something here? Like unexpected touch phase $phase received. So if we forget anything, we'll get reminded.

Reply:
We are veering away from the adding logs that the user/developer cannot take action on. I'd prefer we track it via the issue instead.

Reply:
Maybe just an FML_DLOG(INFO)?  That wouldn't show up for normal developers.

Reply:
Done.